app/account/ChangePassword/page.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
```

Comment:
if you are going to render the whole page on the client you don't need a separate file for the forms. They were in separate files in mine because the main page layout is rendered server-side